src/classes/acl.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Jambura;
+class Acl
+{
+    private $rules;
+    public static function init()
+    {
+        return new self();
+    }
+
+    // Set the rules as defined in the Controller
+    public function setRules($rules)
+    {
+        $this->rules = $rules;
+
+        return $this; 
+    }
+
+    // Checks whether the action has access to be viewed
+    public function checkAccess($action)
```

Comment:
please change the method name to `canAccess()` - The code may look like this

```
public function canAccess(String $action) : bool
{
    try {
        $accessRule = $this->getRuleForAction($action); // raises jamexNoAclFound  if no rule found for action
        if ($this->isDenied($accessRule, $action)) {
            return false;
        }
        
        if (!$this->isAllowed($accessRule, $action)) {
            return false;
        }

        return true;
    } catch (\jamexNoAclFound $e) {
        return true;
    }
}
````

example, ACL rule - 

```
[
    'login' => ['allow' => 'all'], // Meaning login controller will be allowed to all users 
    'usermanagement' => ['allow' => ['User::hasAccess', ['admin']]], // Will call Users::withAccess('admin')
    'groupmanagement' => ['sameAs' => 'usermanagement'],  // Use the same rule as usermanagement
    'myprofile' => ['allow' => 'Controller_site::isMyProfile'], // Will call Controller_site::isMyProfile() with no params
    '*' => ['deny' => 'all']
]
```